unused/hello_no_macros.rs

```diff
@@ -8,26 +8,26 @@ extern crate redis_module;
 
 use redis_module::raw;
 use redis_module::Context;
-use redis_module::{Command, RedisResult, RedisValue, RedisError};
+use redis_module::{Command, ValkeyResult, ValkeyValue, RedisError};
```

Comment:
I like how there is a file called "unused".
```suggestion
use redis_module::{Command, ValkeyResult, ValkeyValue, ValkeyError};
```

Reply:
it was there before, I did not want to remove it in this PR.